docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,130 @@ weight = 4
+      - Enabled automatically after one minute of user inactivity.
+      - You can enable it manually using the `d v` sequence shortcut, or by appending the parameter `?inactive` to the dashboard URL. 
+      - You can disable it with any mouse mouse movement or keyboard action. 
+   - **TV mode (with auto fit panels):** 
+      - The side menu is hidden/removed. 
+      - The navbar, row and panel controls appear at the top of the screen.
+      - Dashboard panels automatically adjust to optimize space on screen.
+   - **Kiosk mode:** 
+      - The side menu, navbar, row and panel controls are completely hidden/removed from view. 
+      - You can enable it manually using the `d v` sequence shortcut after the playlist has started.
+      - You can disable it manually with the same shortcut.
+   - **Kiosk mode (with auto fit panels):** 
+      - The side menu, navbar, row and panel controls are completely hidden/removed from view. 
+      - Dashboard panels automatically adjust to optimize space on screen.
 
 ## Control a playlist
 
-Playlists can also be manually controlled utilizing the Playlist controls at the top of screen when in Playlist mode.
-
-Click the stop button to stop the Playlist, and exit to the current Dashboard.
-Click the next button to advance to the next Dashboard in the Playlist.
-Click the back button to rewind to the previous Dashboard in the Playlist.
-
-## TV or kiosk mode
-
-In TV mode the top navbar, row and panel controls will all fade to transparent.
-
-This happens automatically after one minute of user inactivity but can also be toggled manually
-with the `d v` sequence shortcut, or by appending the parameter `?inactive` to the dashboard URL. Any mouse movement or keyboard action will
-restore navbar and controls.
+You can control a playlist in **Normal** or **TV** mode after it's started, using the navigation bar at the top of your screen.
 
-Another feature is the kiosk mode - in kiosk mode the navbar is completely hidden/removed from view. This can be enabled with the `d k`
-shortcut.
+| Button | Result |
+| --- | --- |
+| Next (double-right arrow) | Advances to the next dashboard. |
+| Back (left arrow) | Returns to the previous dashboard. |
+| Stop (square) | End the playlist, and exits to the current dashboard. |
+| Cycle view mode (monitor icon) | Rotates the display of the dashboards in different view modes. |
+| Time range | Displays data within a time range. It can be set to display the last 5 minutes up to 5 years ago, or a custom time range, using the down arrow. |
+| Refresh (circle arrow) | Reloads the dashboard, to display the current data. It can be set to reload automatically every 5 seconds to 1 day, using the drop down arrow. |
 
-To put a playlist into kiosk mode, use the `d k` shortcut after the playlist has started. The same shortcut will toggle the playlist out of kiosk mode.
+>Shortcut: Click the Esc key to stop the playlist from your keyboard.
```

Comment:
```suggestion
> Shortcut: Press the Esc key to stop the playlist from your keyboard.
```

I think "Click" is for mouse actions only?

Reply:
Yes, and "press" is better according to the Microsoft Style Guide. Thanks!

Click: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click
Press: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/press-and-hold